library/scrollphathd/api/api.py

```diff
@@ -0,0 +1,73 @@
+from sys import exit
+import scrollphathd
+from scrollphathd.fonts import font3x5
+from queue import Queue
+from threading import Thread
+from .action import Action
+from http import HTTPStatus
+
+try:
+    from flask import Blueprint, render_template, abort, request
+except ImportError:
+    exit("flask must be installed in order to use the api. Install with pip install flask")
```

Comment:
I've started replacing `exit` with `raise ImportError()` across all of our libraries. Turns out just calling `exit` is really bad for a lot of reasons! ( who knew! ) I only recently (after years of using it I'm sure) realised it just causes IDLE to abort the script and not print out any useful information at all.